README.md

```diff
@@ -1,2 +1,80 @@
-Ansible Collection: community.aws
-=================================================
+# Community AWS Collection
+[![Shippable build status](https://api.shippable.com/projects//5e5ed2ae0fcc0d0006d2c037badge?branch=master)](https://api.shippable.com/projects/i5e5ed2ae0fcc0d0006d2c037/badge?branch=master)
+<!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/community.aws)](https://codecov.io/gh/ansible-collections/community.aws)-->
+
+The Ansible Community AWS collection includes a variety of Ansible content to help automate the management of AWS instances. This collection is maintained by the Ansible community.
+
+## Included content
+
+Click the ``Content`` button to see the list of content included in this collection.
+
+## Installing this collection
+
+You can install the AWS collection with the Ansible Galaxy CLI:
+
+    ansible-galaxy collection install community.aws
+
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: community.aws
+    version: 0.1.1
+```
+## Using this collection
+
+
+You can either call modules by their Fully Qualified Collection Namespace (FQCN), such as `community.aws.ec2_instance`, or you can call modules by their short name if you list the `community.aws` collection in the playbook's `collections` keyword:
+
+```yaml
+---
+  - name: Create a DB instance using the default AWS KMS encryption key
+    community.aws.rds_instance:
+      id: test-encrypted-db
+      state: present
+      engine: mariadb
+      storage_encrypted: True
+      db_instance_class: db.t2.medium
+      username: "{{ username }}"
+      password: "{{ password }}"
+      allocated_storage: "{{ allocated_storage }}"
+
+```
+
+
+### See Also:
+
+* [Amazon Web Services Guide](https://docs.ansible.com/ansible/latest/scenario_guides/guide_aws.html)
+* [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) for more details.
+
+## Contributing to this collection
+
```

Comment:
I think this section should link to CONTRIBUTING.md with contributing specifics documented in that file.